pkg/windowsdesktop/sfx/Directory.Build.props

```diff
@@ -19,25 +19,19 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.DotNet.Wpf.GitHub" Version="$(MicrosoftDotNetWpfGitHubVersion)" />
     <PackageReference Include="Microsoft.Private.Winforms" Version="$(MicrosoftPrivateWinformsVersion)" GeneratePathProperty="true" />
-    <PackageReference Include="Microsoft.Win32.Registry.AccessControl" Version="$(MicrosoftWin32RegistryAccessControlVersion)" />
```

Comment:
Microsoft.Win32.Registry.AccessControl isn't part of the `Microsoft.NETCore.App.Ref` targeting pack. Should you don't want to expose it in the windowsdesktop targeting pack anymore? That would be a breaking change.